nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/OperationChainContribution.java

```diff
@@ -207,16 +208,16 @@ public OperationChain toOperationChain(Bundle bundle) throws OperationException
                     case 'd':
                         if (T_DOCUMENT.equals(type)) {
                             if (param.value.startsWith(".")) {
-                                val = Scripting
-                                        .newExpression("Document" + ".resolvePathAsRef(\"" + param.value + "\")");
+                                val = Scripting.newExpression(
+                                        "Document" + ".resolvePathAsRef(\"" + param.value + "\")");
```

Comment:
Check if we can't also put this code in common with other (but it's slightly different...)